Gems/Atom/Tools/MaterialCanvas/Assets/MaterialCanvas/TestData/mad.materialcanvasnode.azasset

```diff
@@ -0,0 +1,105 @@
+{
+    "Type": "JsonSerialization",
+    "Version": 1,
+    "ClassName": "DynamicNodeConfig",
+    "ClassData": {
+        "category": "Math Functions",
+        "title": "mad",
+        "inputSlots": [
+            {
+                "name": "inValue1",
```

Comment:
Suggest names like `scale`, `value`, and `bias` or something along those lines. `mvalue`, `avalue`, and `bvalue` also work.